### DISCUSSION TITLE
Target System 没有QEMU ARM Virtual Machine 选项

### DISCUSSION THREAD

**Discussion by Black0Bag** [coolsnowwolf/lede#11342] [Bugs / 问题反馈]
求大神解答，是我把源码搞乱了吗？
在Target System 里没有QEMU ARM Virtual Machine 选项，想要编译无从下手。
![屏幕截图 2023-07-02 154023](https://github.com/coolsnowwolf/lede/assets/22141039/0d7cbfc3-72b2-4d93-9953-09e28d0a4f3a)

**Comment by WYC-2020** [comment]
有
![image](https://github.com/coolsnowwolf/lede/assets/34025388/bcb17271-fb72-4d55-8ac5-8c4911911253)

**Comment by banmuyutian** [comment]
我也遇到了相同的问题，解决了吗？

**Comment by Kilomiyaa** [comment]
有解决方法吗？

**Comment by copybyte** [comment]
好像最新的代码又没了
